ipinfo/cmd_init.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/spf13/pflag"
+	"golang.org/x/term"
+)
+
+func printHelpInit() {
+	fmt.Printf(
+		`Usage: %s init [<opts>] [<token>]
+
```

Comment:
Can you also add an example (see other commands that provide `Examples:` text) or two? They will be simple of course, but it gives a quick idea of how to call it with and without the token. No need to show the responses that will come from that.